src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -247,6 +247,10 @@ def load_command_table(self, _):
         g.custom_command('config', 'enable_cd')
         g.custom_command('show-cd-url', 'show_container_cd_url')
 
+    with self.command_group('webapp deployment github-actions') as g:
```

Comment:
what about adding preview tag for one or two sprint to test the functionality for new commands?